dashboard/test/controllers/maker_controller_test.rb

```diff
@@ -4,11 +4,48 @@ class MakerControllerTest < ActionController::TestCase
   include Devise::Test::ControllerHelpers
 
   setup do
+    @student = create :student
     @teacher = create :teacher
     @admin = create :admin
     @school = create :school
   end
 
+  test "home redirects to sign-in when user is signed out" do
```

Comment:
You can on-line this with `test_redirect_to_sign_in_for` (defined in test_helper.rb by Andrew)